packages/server/src/games.ts

```diff
@@ -53,8 +59,14 @@ function calculatePlayerAction(oldState: GameState, playerAction: PlayerActionPl
     ];
     newState.table = newState.table.filter((c) => !tableCards.includes(cardKey(c)));
     newState.latestCaptured = activePlayer.username;
+    const cardsText = playerAction.tableCards.reduce(
+      (text, value, i, array) => `${text}${i < array.length - 1 ? ', ' : ' and '}${getCardName(value)}`,
```

Comment:
doesn't this place a comma before the first card?

Reply:
No
![image](https://user-images.githubusercontent.com/908935/104090685-545b0c80-5270-11eb-85ad-0ce5b6c609b0.png)